src/components/RoomSelector.vue

```diff
@@ -178,7 +177,8 @@ export default {
 				})
 
 			this.rooms = response.data.ocs.data.sort(this.sortConversations)
-				.filter(conversation => !hasTalkFeature(this.currentRoom, 'federation-v1') || !conversation.remoteServer)
+				// TODO MessageForwarder should work
+				.filter(conversation => !conversation.remoteServer) // no attachments support in federated conversations
```

Comment:
attachments?

Reply:
RoomSelector is used in Deck integration, that's why it was filtered out (to not send cards)